Include/internal/pycore_object.h

```diff
@@ -32,6 +32,16 @@ PyAPI_FUNC(void) _Py_NO_RETURN _Py_FatalRefcountErrorFunc(
 #define _Py_FatalRefcountError(message) \
     _Py_FatalRefcountErrorFunc(__func__, (message))
 
+// Increment reference counter by n
```

Comment:
```suggestion
// Increment reference count by n
```

Also, maybe "increase" instead of "increment"